nautobot/ipam/filters.py

```diff
@@ -237,10 +237,11 @@ class PrefixFilterSet(
         to_field_name="name",
         label="Namespace (name or ID)",
     )
+    ip_version = django_filters.NumberFilter()
```

Comment:
This change is needed because it is broken in `DynamicGroupEditView`. The error says that it is expecting a list of values instead of a single value. I do not know why the filter form is not broken in `IPAddressListView` perhaps `DynamicGroup` has a `filter_form.is_valid():` call in its `post()` method.